MapboxCoreNavigation/NavigationSettings.swift

```diff
@@ -23,10 +23,28 @@ public class NavigationSettings: NSObject {
     @objc public dynamic var voiceVolume    : Float     = 1.0
     
     /**
-     Indicates whether the voice controller should be muted or not.
+     Specifies whether to mute the voice controller or not.
      */
     @objc public dynamic var voiceMuted     : Bool      = false
     
+    /**
+     Specifies the preferred distance measurement unit.
+     - note: Anything but `kilometer` and `mile` will fall back to the default measurement for the current locale.
+        Meters and feets will be used when the presented distances are small enough. See `DistanceFormatter` for more information.
+     */
+    @objc public dynamic var distanceUnit : LengthFormatter.Unit = Locale.current.usesMetric ? .kilometer : .mile
+    
+    var usesMetric: Bool {
+        get {
+            if distanceUnit == .kilometer {
```

Comment:
I would of used a `switch` here, but there's nothing wrong with this.

Reply:
Agreed, `switch` would feel better here.